docs/development/core/server/kibana-plugin-core-server.savedobjectsutils.getconvertedobjectid.md

```diff
@@ -0,0 +1,28 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-core-server](./kibana-plugin-core-server.md) &gt; [SavedObjectsUtils](./kibana-plugin-core-server.savedobjectsutils.md) &gt; [getConvertedObjectId](./kibana-plugin-core-server.savedobjectsutils.getconvertedobjectid.md)
+
+## SavedObjectsUtils.getConvertedObjectId() method
+
+Uses a single-namespace object's "legacy ID" to determine what its new ID will be after it is converted to a multi-namespace type.
+
+<b>Signature:</b>
+
+```typescript
+static getConvertedObjectId(namespace: string | undefined, type: string, id: string): string;
+```
+
+## Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  namespace | <code>string &#124; undefined</code> |  |
+|  type | <code>string</code> |  |
+|  id | <code>string</code> |  |
+
+<b>Returns:</b>
+
```

Comment:
I wonder why the doc generator didn't pick up the parameter descriptions or the return description 🤔

Reply:
A couple ideas:
- The script doesn't regen the docs if the signature didn't change, so if you add the comments on the params after generating this the first time it may not be populated. You can add the `--docs` CLI arg to force this
- I wonder if the `{string}` type descriptors are a problem? I usually don't use these since that info is already in the TS signature itself. Not sure how our doc generator handles these.
- It may not matter much, we're about to remove this tooling.